lib/util.js

```diff
@@ -21,20 +21,20 @@ module.exports.exec = function(command, opts, cb) {
         exitCode: error && error.code,
         exitSignal: error && error.signal
       });
+      return;
```

Comment:
Added this `return` to help `eslint`. This is how it fixed without it, I think the comment is interfering from recognizing `else if`.

```js
if (opts.handleError && (error || stderr)) {
  cb({
    stdout: stdout,
    stderr: [stderr, error].filter(v => !!v).join(''),
    exitCode: error && error.code,
    exitSignal: error && error.signal
  });
} else {
  // if (stdout) console.log(stdout);
  if (stderr) {
    if (cb.length == 1) cb(stderr);
    else throw new Error(stderr);
  }
  else if (error !== null) {
    if (cb.length == 1) cb(error, stdout, stderr);
    else throw error;
  }
  else {
    cb();
  }
}
```